DTS-STN/dts-rmp
# Header/footer

# Description

Added:
- Header (As is in the mock-up. Will probably change)
- Footer
- Used the language toggle for all images and links.

## Test Instructions

1. Load the page
2. Test the links to see that they bring you to the right website.
3. Test the toggle to see that the language changes for all links and images (including alt and title tags) 

## Help Requested

Something to notice:
- At 320px and lower, the footer 'Go to top page' might wrap a little bit. Is this a problem?

## Checklist
